templates/miq-template-ext-db.yaml

```diff
@@ -192,6 +192,108 @@ objects:
           resources:
             requests:
               storage: "${APPLICATION_VOLUME_CAPACITY}"
+
+
+- apiVersion: apps/v1beta1
+  kind: "StatefulSet"
+  metadata:
+    name: ${NAME}-backend
```

Comment:
"engine" to me suggests that it is a required service, I don't have any better suggestions either.